ibtk/include/ibtk/FEDataManager.h

```diff
@@ -1129,10 +1145,29 @@ class FEDataManager : public SAMRAI::tbox::Serializable
      * are allowed to move no more than one cell width between regridding
      * operations).
      *
+     * The parameters refer to the levels of different objects:
+     * <ol>
+     *   <li>@p level_number - the level number in the patch hierarchy on which
+     *     we are identifying intersections.</li>
+     *   <li>@p coarsest_elem_ln - The minimum level number of elements we should
```

Comment:
For input parameters, I wonder if it is better to be more verbose, e.g.:
```suggestion
     *   <li>@p coarsest_elem_level_num - The minimum level number of elements we should
```